includes/results.inc

```diff
@@ -854,6 +854,17 @@ class IslandoraSolrFacets {
   }
 
   /**
+   *
+   * @return mixed
```

Comment:
seems like codeSniffer is expecting a comment after/partof @return statement?  Kinda new for me, or i'm just getting old.